src/index.ts

```diff
@@ -48,16 +52,18 @@ function startObservingComments() {
 // Create a sticking wrapper to place all reactions
 function injectWrapper() {
   // console.log('injectWrapper')
-  const header = document.querySelector(sideBarId)
+  const header = document.querySelector(sideBarId) as HTMLDivElement
```

Comment:
Not a big deal, but document.querySelector<HTMLDivElement> is safer because it's asking if it can be a Div, not telling it